dind-provider-cli/Dockerfile

```diff
@@ -1,24 +1,33 @@
-FROM docker:18.06.3-ce-dind
+FROM docker:19.03.8-dind
 LABEL maintainer="Rémi Marseille <marseille@ekino.com>"
 
 ARG CI_HELPER_VERSION
 ARG DOCKER_COMPOSE_VERSION
 ARG GLIBC_VERSION
 ARG PIPENV_VERSION
 ARG PIP_VERSION
+ARG IBM_CLI_VERSION
+ARG AZURE_CLI_VERSION
 ENV PYTHON_PIP_VERSION ${PIP_VERSION}
 
 RUN echo "Install AWS" && \
-    apk add -q --no-cache bash build-base ca-certificates curl gettext git libffi-dev linux-headers make musl-dev openldap-dev openssh-client python py-pip rsync tzdata && \
-    pip install pip==${PYTHON_PIP_VERSION} && \
+    apk add -q --no-cache bash build-base ca-certificates curl gettext git libffi-dev linux-headers musl-dev openldap-dev openssh-client python3-dev gcc libffi-dev libressl-dev make rsync tzdata groff && \
+    python3 -m ensurepip && \
+    pip3 install --no-cache --upgrade pip setuptools wheel && \
     pip -q install pipenv==${PIPENV_VERSION} awscli boto3 PyYAML && \
     echo "Done install AWS" && \
 
-    echo "Install CI Helper" && \
```

Comment:
Please do not remove the CI Helper. We still need it.

Reply:
Fixed